obspy/taup/tau_model.py

```diff
@@ -413,3 +371,206 @@ def getBranchDepths(self):
         branchDepths += [self.getTauBranch(
             i - 1, True).botDepth for i in range(1, len(self.tauBranches[0]))]
         return branchDepths
+
+    def serialize(self, filename):
+        """
+        Serialize model to numpy npz binary file.
+
+        Summary of contents that have to be handled during serialization::
+
+            TauModel
+            ========
+            cmbBranch <type 'int'>
+            cmbDepth <type 'float'>
+            debug <type 'bool'>
+            iocbBranch <type 'int'>
+            iocbDepth <type 'float'>
+            mohoBranch <type 'int'>
+            mohoDepth <type 'float'>
+            noDisconDepths <type 'list'> (of float!?)
```

Comment:
Oh, actually, `splitBranch` is called when the model is loaded and a source depth is set. So as long as the results don't change there...